src/Fetch_Command.php

```diff
@@ -156,22 +161,25 @@ public function __invoke( $args, $assoc_args ) {
 			}
 		}
 
-		$skip_cache = Utils\get_flag_value( $assoc_args, 'skip-cache' );
-		if ( $skip_cache ) {
+		if ( Utils\get_flag_value( $assoc_args, 'skip-sanitation' ) ) {
+			remove_filter( 'oembed_dataparse', 'wp_filter_oembed_result', 10, 3 );
+		}
+
+		if ( Utils\get_flag_value( $assoc_args, 'skip-cache' ) ) {
 			$wp_embed->usecache = false;
 			// In order to skip caching, also need `$cached_recently` to be false in `WP_Embed::shortcode()`, so set TTL to zero.
 			add_filter( 'oembed_ttl', '__return_zero' );
 		} else {
 			$wp_embed->usecache = true;
 		}
 
-		if ( ! $discover ) {
-			// `WP_Embed::shortcode()` sets the 'discover' attribute based on this filter, no matter what's passed to it, so set to false.
-			add_filter( 'embed_oembed_discover', '__return_false' );
-		}
+		// `WP_Embed::shortcode()` sets the 'discover' attribute based on 'embed_oembed_discover' filter, no matter what's passed to it.
+		add_filter( 'embed_oembed_discover', $discover ? '__return_true' : '__return_false' );
```

Comment:
Would it make sense to remove all existing filters (from plugins & themes)  prior to adding our setting?

Reply:
Yes, I think it would, on all the filters added here - think maybe add with priority `PHP_INT_MAX` (as here https://github.com/wp-cli/media-command/blob/v1.1.3/src/Media_Command.php#L841), which would always win so has the same effect and saves having to put in `remove_all_filters()`  in front of each?

Reply:
I pushed using `PHP_INT_MAX` - see what you think...

Reply:
Thinking about it might as well remove the filters for `oembed_dataparse` at least as it's the only one with default filters so saves some needless processing.